docs/source/onnx.rst

```diff
@@ -0,0 +1,137 @@
+      onnx_seq = ONNXSequential("model1.onnx", "model2.onnx", io_map=io_map)
+
+4. **Exporting the Combined Model**
+
+   You can easily export the combined model to an ONNX file:
+
+   .. code-block:: python
+
+      # Export the combined model to a file
+      onnx_seq.export("combined_model.onnx")
+
+5. **Optimizing with Execution Providers**
+
+   Leverage ONNXRuntime's execution providers for optimized inference. For example, to run the model on a GPU:
+
+   .. code-block:: python
+
+      # Initialize with CUDA execution provider
+      onnx_seq = ONNXSequential(
+         "hf://operators/kornia.geometry.transform.flips.Hflip",
+         # Or you may use a local model with either a filepath "YOUR_OWN_MODEL.onnx" or a loaded ONNX model.
+         "hf://models/kornia.models.detection.rtdetr_r18vd_640x640",
+         providers=['CUDAExecutionProvider']
+      )
+
+      # Run inference
+      outputs = onnx_seq(input_data)
+
+
+Frequently Asked Questions (FAQ)
+--------------------------------
+
+**1. Can I chain models from different sources?**
+
+Yes! You can chain models from different ONNX files or directly from `onnx.ModelProto` objects. `ONNXSequential` handles the integration and merging of their graphs.
+
+**2. What happens if the input/output sizes of models don't match?**
+
+You can use the `io_map` parameter to control how outputs of one model are mapped to the inputs of the next. This allows for greater flexibility when chaining models with different architectures.
+
+**3. Can I use custom ONNXRuntime session options?**
+
+Absolutely! You can pass your own session options to the `create_session` method to fine-tune performance, memory usage, or logging.
+
+Why Choose ONNXSequential?
+--------------------------
+
+With the increasing adoption of ONNX for model interoperability and deployment, `ONNXSequential` provides a simple yet powerful interface for combining models and operators. By leveraging ONNXRuntime’s optimization and execution provider capabilities, it gives you the flexibility to:
+- Deploy on different hardware (CPU, GPU).
+- Run complex pipelines in production environments.
+- Combine and experiment with models effortlessly.
+
+Whether you're building an advanced deep learning pipeline or simply trying to chain pre-trained models, `ONNXSequential` makes it easy to manage, optimize, and execute ONNX models at scale.
+
+Get started today and streamline your ONNX workflows!
+
+
+API Documentation
+-----------------
+.. autoclass:: kornia.onnx.sequential.ONNXSequential
+    :members:
+
+.. autoclass:: kornia.onnx.utils.ONNXLoader
+
+    .. code-block:: python
+
+        onnx_loader = ONNXLoader()
+        # Load a HuggingFace operator
+        onnx_loader.load_model("hf://operators/kornia.color.gray.GrayscaleToRgb")  # doctest: +SKIP
+        # Load a local converted/downloaded operator
+        onnx_loader.load_model("operators/kornia.color.gray.GrayscaleToRgb")  # doctest: +SKIP
```

Comment:
We can here to show case remap ? It will be a very valuable function if it’s well supported in ONNX with all the providers, eg for self driving car industry and in general spatial ai/robotics